build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
i dont see any of those dependencies being used here!All i am saying is keep it simple